src/components/command-bar/components/activator/index.tsx

```diff
@@ -8,19 +11,37 @@ const CommandBarActivator = ({
 	visualLabel,
 }: CommandBarActivatorProps) => {
 	const { setIsOpen } = useCommandBar()
+	const isMacOs = getIsUserAgentMacOs()
```

Comment:
🤔 I believe this has the potential to cause hydration errors, as the initial markup will always be rendered with this as `false`

Reply:
Want to render 2 icons and do our usual CSS magic to show/hide one of them?

Reply:
I think that would be ideal! If it's possible to detect windows via CSS? Otherwise maybe we could render something that has both icons?

![Screen Shot 2022-09-07 at 3 20 34 PM](https://user-images.githubusercontent.com/1289701/188969949-992f093d-71c9-4cc1-a217-ad81a845f3ec.png)

Reply:
Updated! The CSS magic didn't work because JavaScript was still needed to detect the OS. I also tried the idea in your screenshot, but the `/` isn't vertically aligning like regular characters. Went with just the `command-16` icon and have added a note to the PR description.